megamek/src/megamek/common/Tank.java

```diff
@@ -3984,7 +4085,8 @@ public boolean isDmgLight() {
 
         return ((double) totalInoperable / totalWeapons) >= 0.25;
     }
-
+    
+    @Override
```

Comment:
There's no need to override this to do return the same value returned by this method in the parent class.

Reply:
Fixed. I was thinking I had to Override Tank to Override SupportTank or SuperHeavyTank